Follow the code review recommendations

xandro2021 opened the pull request:
# Set Linters and follow the code reviewers' recommendations
in this PR:
+ Add linters
+ Style the title to see if the linters are working properly
+ Update README.md
+ Add LICENSE.md
+ Follow the code reviewer's recommendations

Thank you for review my PR

xandro2021 committed:
Merge pull request #8 from MyMicroverseProjects/main

Merge pull request #7 from MyMicroverseProjects/set-linters

xandro2021 committed:
add the title a wavy underline style

xandro2021 commented:
All the evaluations on github went well, but in terminal it was an warning message when I executed npx hint .
![screen](https://user-images.githubusercontent.com/88587370/218905595-eeefc666-55b7-445c-afee-0cacfa03e62f.png)
![htmlerror](https://user-images.githubusercontent.com/88587370/218905633-cc21bd3b-83da-4331-9f39-0f38cac87626.png)
![ErrorHtml](https://user-images.githubusercontent.com/88587370/218905648-ee72d64d-0fc0-4c02-bceb-0ee7bcf926f5.png)

xandro2021 commented:
Why is there a warning error like if it was trying to parsing the index.html as JSON file?
![htmlerror](https://user-images.githubusercontent.com/88587370/218905954-59516bdc-6c15-4911-9849-3d78ff289375.png)

AbayomiOlaoye reviewed (changes requested):
## STATUS: CHANGE REQUIRED ♻️ 

Hi @xandro2021 :wave: , 

Good job with this milestone so far!
There are just few more issues to work on before moving onto the next project but you are almost there!

![so_close](https://user-images.githubusercontent.com/99715640/211950260-0e3881c0-9d4f-4ea2-a670-9ef45a23a0f5.gif)

## Highlights
- No linter error :heavy_check_mark: 
- Good implementation :heavy_check_mark: 

## Required Changes ♻️

_Check the comments under the review._(All comments will be here in since I do not have access to the codes in question but I have tried to get explicitly edited screenshots to suggest how you can improve on your code and project submission.

- [ ] A very important best practice is documenting your work properly, from code to comments, readme, and pull requests. Please look at the image below and keep in mind how descriptive your Pull request message should be.

![image](https://user-images.githubusercontent.com/99715640/218907821-7e5ad343-d3b1-44a5-a1b3-4b9ffe2e1d3f.png)

  -  [ ] A good point to note here is that `styles` were not all that this project requirement focused on but due to improper use of `Gitflow` you seem to miss the goodies :smile: Please always remember that requirements or user stories must be done in a single, and this is where you good commit will help you in case you need to revisit past snapshots.

- [ ] Below is a snapshot of your ReadMe file and some suggested changes:

![image](https://user-images.githubusercontent.com/99715640/218910295-49723347-b0bc-4668-8835-95b5397edfb2.png)

- You are almost done with this project, @xandro2021 :wink: 
- - [ ] :+1: Like I said, you have made a brilliant effort to get this up and running you still need to give your nice README.md some more time and attention so that you can have contents that make your project look professional to visitors or users.
  - [ ] * The only **optional** section is FAQ.
  * You can also delete the live demo section since you are not deploying your application now or you just let visitors know that it is not ready by writing `N/A` - Not Available instead of adding an irrelevant link. That will improve your user experience,
  * Kindly add more descriptive message to **About your project**.
  * Also, kindly include the 2-3 **Future features** of the project in the future project instead of the template input there. This doesn't add any value to your visitors and it is not a professional practice to claim code that lacks customization.
* It will be more customized to **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
* Your `LICENSE` link is not working at the moment and I think that is due to the absence of `LICENSE` file in your root folder. Kindly add it and ensure it leads to the appropriate resource. This offers valuable information to your users about how to use or share your intellectual work.
* In Summary, kindly remove all the elements in read box

Keep doing an amazing job, Micronaut! :clap: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Create LICENSE.md

xandro2021 committed:
update README.md according with code reviewer recomendation

1) +1 Like I said, you have made a brilliant effort to get this up and running you still need to give your nice README.md some more time and attention so that you can have contents that make your project look professional to visitors or users.
2) * The only optional section is FAQ.
3) You can also delete the live demo section since you are not deploying your application now or you just let visitors know that it is not ready by writing N/A - Not Available instead of adding an irrelevant link. That will improve your user experience,
4) Kindly add more descriptive message to About your project.
5) Also, kindly include the 2-3 Future features of the project in the future project instead of the template input there. This doesn't add any value to your visitors and it is not a professional practice to claim code that lacks customization.
6) It will be more customized to remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
7) Your LICENSE link is not working at the moment and I think that is due to the absence of LICENSE file in your root folder. Kindly add it and ensure it leads to the appropriate resource. This offers valuable information to your users about how to use or share your intellectual work.
8) In Summary, kindly remove all the elements in read box

xandro2021 commented:
I cannot change the branch name otherwise Github will automatically close this pull request because their system to rename a branch consist in delete the branch to create a new copy of the branch with the new name.
Microverset doesn't allow me to open a new pull request for this project, so I cannot change the branch name.
This pull request received a new commit where a tried to improve in the other aspects

xandro2021 committed:
Fix licence link in README.md

+ It was change thed link pointing to the license because it wasn't
  working with the default configuration of the README.md provided by
  Microverse

aamirkhan2478 commented on README.md:
```diff
@@ -184,32 +160,23 @@ Example:
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Mastodon: [Mastodon](https://uiuxdev.social/@xandrocastro)
```
- [ ] Kindly add your social media handles here. :arrow_double_up:

xandro2021 replied:
I couldn't understand your optional suggestion. All my social media links are working properly there, including the extra one from Mastodon. Then, I just add my name in the Author1 Title

xandro2021 replied:
Ok, I change the pull request description as you have suggested

aamirkhan2478 reviewed (changes requested):
Hi @xandro2021 , 

![](https://media4.giphy.com/media/CoND5j6Bn1QZUgm1xX/giphy.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ No linter errors

 ## Required Changes ♻️
-  [ ] Please add a descriptive PR description as mentioned by the previous reviewer.

- Your description:
![Screenshot from 2023-02-16 02-24-26](https://user-images.githubusercontent.com/45226848/219171630-e1adeb49-0cc0-4dd7-a38a-2cbbbb25e612.png)

- Example Description:
![Description-image](https://user-images.githubusercontent.com/87197412/184990532-da778ab4-c789-4136-90a9-629eedd969da.png)

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Add authors name in README.md (update)

+ Update PR Description
+ Add your social media handles here. arrow_double_up

victorokoroji reviewed (approved):
Hi @xandro2021, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
